CONTRIBUTING.md

```diff
@@ -23,6 +23,10 @@ Please take a look at this list first, your contributions may belong in one of t
    programmatic instrumentations that are now in the `ext` directory in the main OpenTelemetry repo. Please ask in the Gitter
    channel (see below) for guidance if you want to contribute with these instrumentations.
 
+# Find the right branch
+
+The default branch for this repo is `main`. Changes that pertain to components marked as `stable` in the [specifications](https://github.com/open-telemetry/opentelemetry-specification) go into this branch, which currently does not include `metrics`. Changes that pertain to `metrics` go into the `metrics` branch.
```

Comment:
Should that branch be named `metrics`? Or should we use a more general-purpose name because maybe some other unstable things show up that are not metrics :shrug:

Reply:
Good question. We do not want the branch to have "everything that is unstable". Metrics is the only thing that is going to be "long-lived" in terms of development and so it is appropriate to isolate that into its own branch. As well from a maintenance perspective, if we have a development branch, everytime we want to release stable changes, we would have to cherrypick from the development branch into the main branch, because we do not want to include the metrics changes as well as part of the release (assuming metrics has ongoing, long-lived development).